src/Params/EnvelopeParams.cpp

```diff
@@ -496,8 +555,15 @@ void EnvelopeParams::getfromXML(XMLwrapper& xml)
     for(int i = 0; i < Penvpoints; ++i) {
         if(xml.enterbranch("POINT", i) == 0)
             continue;
-        if(i != 0)
-            Penvdt[i] = xml.getpar127("dt", Penvdt[i]);
+        if(i != 0) {
+            if(!xml.hasparreal("A_dt")) {
```

Comment:
This seems to be the only actual bug. It should be xml.hasparreal("dt") unless I'm mistaken.